otherlibs/win32unix/readlink.c

```diff
@@ -0,0 +1,104 @@
+/***********************************************************************/
+/*                                                                     */
+/*                                OCaml                                */
+/*                                                                     */
+/*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         */
```

Comment:
Ah, OK - no, all the code's mine, beyond lifting the GC boilerplate macros from the UNIX version (which is how the header remained). I'll push amendments in a few mins...